.github/workflows/deployAzureFrontend.yml

```diff
@@ -26,11 +26,11 @@ jobs:
         with:
           node-version: "14"
       - name: Install dependencies
-        run: npm install
+        run: yarn install
       - name: Build deploy artifact
-        run: npm run build
+        run: yarn run build
```

Comment:
Is the `run` necessary/helpful?

Reply:
I think the `yarn` convention is not to use `run`. I think you might be able to name a script the same as a yarn command and `run` allows you to specify the script